src/components/SearchResults.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import moment from "moment";
+
+export class TableRow extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false,
+      style: {}
+    };
+  }
+
+  onClicked = () => {
+    this.setState(() => {
+      if (this.state.isClicked === false)
```

Comment:
This is correct. However, another way would be to use a `previousState` parameter in the function you use in `setState`. So you can write `previousState.isClicked` instead of `this.state.previousState`. It's generally considered a better practice.